{
    "n_steps": 0,
    "cum_reward": 0,
    "cum_raw_reward": 0,
    "err_msg": "Exception uncaught by agent or environment in task webarena.795.\nTimeoutError:\nPage.goto: Timeout 30000ms exceeded.\nCall log:\n  - navigating to \"http://111.229.174.217:18082/\", waiting until \"load\"\n",
    "stack_trace": "Traceback (most recent call last):\n  File \"/srv/home/weichenzhang/hallucination/BG_for_hallucination/browsergym/experiments/src/browsergym/experiments/loop.py\", line 410, in run\n    step_info.from_reset(\n  File \"/srv/home/weichenzhang/hallucination/BG_for_hallucination/browsergym/experiments/src/browsergym/experiments/loop.py\", line 215, in from_reset\n    self.obs, env_info = env.reset(seed=seed)\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/gymnasium/wrappers/common.py\", line 146, in reset\n    return super().reset(seed=seed, options=options)\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/gymnasium/core.py\", line 333, in reset\n    return self.env.reset(seed=seed, options=options)\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/gymnasium/wrappers/common.py\", line 400, in reset\n    return super().reset(seed=seed, options=options)\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/gymnasium/core.py\", line 333, in reset\n    return self.env.reset(seed=seed, options=options)\n  File \"/srv/home/weichenzhang/hallucination/BG_for_hallucination/browsergym/core/src/browsergym/core/env.py\", line 303, in reset\n    task_goal, task_info = self.task.setup(page=self.page)\n  File \"/srv/home/weichenzhang/hallucination/BG_for_hallucination/browsergym/webarena/src/browsergym/webarena/task.py\", line 130, in setup\n    page.goto(url)\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/playwright/sync_api/_generated.py\", line 9020, in goto\n    self._sync(\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/playwright/_impl/_sync_base.py\", line 115, in _sync\n    return task.result()\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/playwright/_impl/_page.py\", line 552, in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/playwright/_impl/_frame.py\", line 145, in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/playwright/_impl/_connection.py\", line 61, in send\n    return await self._connection.wrap_api_call(\n  File \"/home/weichenzhang/miniconda3/envs/hallu/lib/python3.10/site-packages/playwright/_impl/_connection.py\", line 528, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nplaywright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nCall log:\n  - navigating to \"http://111.229.174.217:18082/\", waiting until \"load\"\n\n",
    "stats.cum_steps": 1,
    "terminated": null,
    "truncated": null
}